test/fakeintake/aggregator/common.go

```diff
@@ -78,6 +78,14 @@ func (agg *Aggregator[P]) ContainsPayloadNameAndTags(name string, tags []string)
 	return false
 }
 
+func (agg *Aggregator[P]) GetNames() []string {
```

Comment:
Where this function is used?
Depending on how it will be used, it might be interesting to sort the result by name as the order of the map is not deterministic.

Reply:
I used it in tests, while writing them, to get a list of all payloads received. For example, it was hard to see what `logs` use for name. Ok for sorting them, good idea.